src/sentry/event_manager.py

```diff
@@ -431,13 +432,13 @@ def _get_event_instance(self, project_id=None):
         date = datetime.fromtimestamp(recorded_timestamp)
         date = date.replace(tzinfo=timezone.utc)
         time_spent = data.get("time_spent")
-
-        data["node_id"] = Event.generate_node_id(project_id, event_id)
-
+        node_data = NodeData(
+            None, None, data=EventDict(data, skip_renormalization=True), wrapper=EventDict
```

Comment:
Isn't this calling the EventDict wrapper passing an object that is already an EventDict ?

Reply:
Looking back at https://github.com/getsentry/sentry/commit/54764bf9d9ed8473784fbecbb86badeaaad10d97 I believe it's needed to renormalize node data like we do with SnubaEvent